src/autocomplete/src/Autocomplete.js

```diff
@@ -153,13 +155,13 @@ const Autocomplete = memo(
           selectedItem,
           ...restDownshiftProps
         }) => (
-          <div style={{ width: '100%' }}>
+          <div style={containerStyle}>
             <Popover
               bringFocusInside={false}
               isShown={isShown}
               minWidth={popoverMinWidth}
               position={position || (targetWidth < popoverMinWidth ? Position.BOTTOM_LEFT : Position.BOTTOM)}
-              content={() => (
```

Comment:
Unnecessary new function created every time. Unless we are using the `close` method we can simply pass the JSX directly